src/ImageSharp/Formats/Jpeg/PdfJsPort/Components/PdfJsScanDecoder.cs

```diff
@@ -13,6 +17,28 @@ namespace SixLabors.ImageSharp.Formats.Jpeg.PdfJsPort.Components
     /// </summary>
     internal struct PdfJsScanDecoder
     {
+        /// <summary>
+        /// Gets the ZigZag scan table
+        /// </summary>
+        private static readonly byte[] DctZigZag =
```

Comment:
You can bring this to the stack, by usiing my `ZigZag` structure + extending it with the `Unsafe.Add()` indexer trick.